### PR TITLE
Update Korean

### PR DESCRIPTION
**Pull request by ZeroOneNebulous**
I changed it to '책임감있는 주인(responsibleowner)' -> '책임감 있는 주인(responsible owner)' in line 1270.

### CODE REVIEW

File: res/values-ko/strings.xml [hunk 1]
```diff
@@ -1266,8 +1266,8 @@ Nebulous 규칙\n
     <string name="x4_Clan_XP_7_days">클랜 경험치 4배: 일주일</string>
     <string name="x4_Clan_XP_1_Day">클랜 경험치 4배: 1일</string>
     <string name="CHARGE_UP">차지 업</string>
-    <string name="CLAN_DESCRIPTION_COLOR">"클랜 설명 색깔"</string>
-    <string name="Responsible_Pet_Owner">책임감있는 주인</string>
+    <string name="CLAN_DESCRIPTION_COLOR">"클랜 설명 색"</string>
```
[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: What changes are you talking about?

[the remark this answers is not in the document]
Comment (Jeii20) [reply res/values-ko/strings.xml:1269]: The `CLAN_DESCRIPTION_COLOR` string

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: <string name="CLAN_DESCRIPTION_COLOR">"클랜 설명 색깔"</string> What's wrong with this part is that you included "(big quotes) unnecessarily when using it as a string.

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: What is wrong with this is that the name of the string resource in the XML resource file does not fit the rule. The name of the string resource should start with a lowercase letter and should not be uppercase. It should also separate the words with an underscore (_).

A good example is:

<string name="clan_description_color">"클랜 설명 색깔"</string>

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: What is wrong with this is that the name of the string resource in the XML resource file does not fit the rule. The name of the string resource should start with a lowercase letter and should not be uppercase. It should also separate the words with an underscore (_).

A good example is : <string name="clan_description_color">"클랜 설명 색깔"</string>

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: What is wrong with this is that the name of the string resource in the XML resource file does not fit the rule. The name of the string resource should start with a lowercase letter and should not be uppercase. It should also separate the words with an underscore (_).

A good example is :

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: <string name="clan_description_color">"클랜 설명 색깔"</string>

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: <string name="clan_description_color">"클랜 설명 색깔"</string>

[the remark this answers is not in the document]
Comment (ZeroOneNebulous) [reply res/values-ko/strings.xml:1269]: image

**Pull request closed without merging**